Sources/tart/VMStorageOCI.swift

```diff
@@ -170,7 +170,7 @@ class VMStorageOCI: PrunableStorage {
 
     if !exists(digestName) {
       let transaction = SentrySDK.startTransaction(name: name.description, operation: "pull", bindToScope: true)
-      let tmpVMDir = try VMDirectory.temporary()
+      let tmpVMDir = try VMDirectory.temporaryDeterministic(key: name.namespace)
```

Comment:
I think name doesn't contain `sha` and can cause a collision. Since we do hashing you can use the full description:

```suggestion
      let tmpVMDir = try VMDirectory.temporaryDeterministic(key: name.description)
```